services/private-gke/client-experimentation/README.md

```diff
@@ -0,0 +1,17 @@
+# GKE Deployment
+
+This folder contains the necessary files to deploy Standard and Autopilot GKE Clusters inside your experimentation landing-zone.
+
+Scripts are being used to build Config Controller Clusters since deploying [Config Controller Instances](https://cloud.google.com/config-connector/docs/reference/resource-docs/configcontroller/configcontrollerinstance) using existing Clusters is still in in alpha and may change without notice. This does not apply to GKE Clusters.
```

Comment:
does this paragraph still applies to this package ? it references config controller which is in a different folder